frontend/src/design/components/SanitizedHTML.js

```diff
@@ -0,0 +1,12 @@
+import DOMPurify from 'dompurify';
+
+export const SanitizedHTML = ({ dirtyHTML }) => {
+  const defaultOptions = {
+    ALLOWED_TAGS: ['b', 'i', 'em', 'strong', 'a'],
+    ALLOWED_ATTR: ['href']
```

Comment:
Are you expecting a link from the config? If not I don't think you should allow `href`. Otherwise, it looks good.

Reply:
@maryamkhidir yes we are using links. That's the main reason I needed HTML support.